### ISSUE TITLE
i2psnark.upbw.max not applied after restart.

### ISSUE THREAD

**Issue by Tunoac** [i2p/i2p.i2p#75]
db52cc7  i2psnark: Bandwidth limiter improvements introduced a bug:
i2psnark.upbw.max is not applied during restart.
Though the desired value is present in i2psnark.config file.
i2psnark.downbw.max is not affected.

I think the bug is here:

SnarkManager.java,  getBWLimit()

```
            int maxup = getInt(PROP_UPBW_MAX, DEFAULT_MAX_UP_BW);
            _util.setMaxUpBW(up);
            _bwManager.setUpBWLimit(Math.min(up, maxup) * 1000L);
```

--> setMaxUpBW is called before the Math.min ?

Thanks.

**Comment by zzzi2p** [comment]
thanks for the report, I'll take a look at it

**Comment by zzzi2p** [comment]
Fixed in 70dfd5155991198e31e3094670e1f6d90a477991 dev 2.5.2-6
Thanks for pointing me to the right place.
This was a display issue only; internal limit was correct.
@eyedeekay please close this ticket as resolved.
